force-app/main/default/classes/DMLOperations.cls

```diff
@@ -160,14 +202,20 @@ public with sharing class DMLOperations {
 	 */
 	public static Account upsertAccount(String accountName) {
 		// Query for the Account
-		List<Account> accounts = [SELECT Id FROM Account WHERE Name = :accountName];
+		List<Account> accounts = [SELECT Id,Name FROM Account WHERE Name = :accountName];
 
 		// if the Account exists, update the description
-
+           if (accounts.size() > 0) {
+            accounts[0].Description = 'Updated Account';
+           } else {
+            Account acc = new Account(Name = accountName, Description = 'New Account');
+            accounts.add(acc);
+           }
 		// otherwise, create a new Account and set the description
 
 		// Upsert the Account
-		return null;
+        upsert accounts;
```

Comment:
Technically here you could have inserted accounts[0]. This would make it clear that you were upserted just one account.
```suggestion
        upsert accounts[0];
```